code/LeftAndMain.php

```diff
@@ -99,7 +99,7 @@ class LeftAndMain extends AdminController implements PermissionProvider
      * @config
      * @var string
      */
-    private static $tree_class = null;
+    private static $model_class = null;
```

Comment:
Most subclasses of `LeftAndMain` which use this config are _not_ trees - and even with `CMSMain` which is the only tree structure, this is getting refactored to be more generic anyway.

Makes sense to update this to be named something that _all_ the classes using it can respect.

Reply:
Is `$model_class` always a `DataObject`? If so this should probably be called `$data_class` instead which as far as I can tell the most common way refer to the "class of the DataObject" i.e. $dataClass.  $model_class seems like it could be any model, including something that's not a DataObject

Reply:
`$model_class` is always a DataObject correct? If so this should probably be called `$data_class` instead which as far as I can tell the most common way refer to the "class of the DataObject" i.e. `$dataClass`. `$model_class` seems like it could be any model, including something that's not a `DataObject`

Reply:
I chose this name to complement `getModelClass()` which currently only exists in `ModelAdmin` but will become more ubiquitous in other cards in the epic. 

In cmsmain and likely others the `getModelClass()` method will return the value held in this config.

If you're okay with `getModelClass()` returning the value of a `data_class` config property I can make the requested change - though that would be a little unintuitive.

Reply:
Yeah, I guess ModelAdmin does use $modelClass everywhere right now, so I guess if that's the local standard we should be using that